backend/schema/protobuf.go

```diff
@@ -132,6 +133,18 @@ func generateStruct(t reflect.Type, messages map[string]string) {
 	messages[t.Name()] = w.String()
 }
 
+// generateWellKnownNestedEnums generates the well-known nested enums for a given reflect.Type.
+// Reflection cannot discover type aliased enums, so they need to get generated manually.
+func generateWellKnownNestedEnums(t reflect.Type, w *strings.Builder) {
```

Comment:
Ah awesome, I've been meaning to do something about enums for a while now.

This, and the "chicken and egg" problem of having the proto schema rely on the Go schema compiling and existing would be nice to solve. There is a more general solution to this whole schema generation thing that I've wanted to do, but it's quite a bit more work, which is to switch to static analysis. But this is great for now.